sources/packages/web/src/services/http/ConfirmationOfEnrollmentApi.ts

```diff
@@ -87,12 +89,12 @@ export class ConfirmationOfEnrollmentApi extends HttpBaseClient {
 
   public async denyConfirmationOfEnrollment(
     locationId: number,
-    applicationId: number,
+    disbursementScheduleId: number,
     denyCOEPayload: DenyConfirmationOfEnrollment,
   ): Promise<void> {
     try {
       await this.apiClient.patch(
-        `institution/location/${locationId}/confirmation-of-enrollment/application/${applicationId}/deny`,
+        `institution/location/${locationId}/confirmation-of-enrollment/disbursement/${disbursementScheduleId}/deny`,
```

Comment:
we have `patchCall`
![image](https://user-images.githubusercontent.com/77353155/153086470-8f1e65b6-1120-46e6-8fff-937b493091d8.png)